fluss-test-coverage/pom.xml

```diff
@@ -347,6 +347,7 @@
                                         <!-- end exclude for lakehouse-paimon -->
                                         <exclude>com.alibaba.fluss.lakehouse.cli.*</exclude>
                                         <exclude>com.alibaba.fluss.kafka.*</exclude>
+                                        <exclde>com.alibaba.fluss.lake.paimon.FlussDataTypeToPaimonDataType</exclde>
```

Comment:
Adding a full types test in `LakeEnabledTableCreateITCase` is better?

Reply:
I add full types of fluss, but we have still to keep it in here since Fluss doesn't support array, map, row type. So the maxinum line coverage can only reach 65%.

Reply:
makes sense to me